pkg/deploy/deployment_keycloak.go

```diff
@@ -80,6 +80,16 @@ func getSpecKeycloakDeployment(checluster *orgv1.CheCluster, clusterDeployment *
 		jbossDir = "/scripts"
 	}
 
+	if clusterDeployment != nil {
+		env := clusterDeployment.Spec.Template.Spec.Containers[0].Env
+		for _, e := range env {
+			if "TRUSTPASS" == e.Name {
+				trustpass = e.Value
```

Comment:
Do you overwrite the global `trustpass` variable ?

Why not having `trustpass` be a local variable in the `getSpecKeycloakDeployment` function then ?

Reply:
The problem with local variable is the following:
- until we have a cluster deployment we will invoke getSpecKeycloakDeployment several times and every time we will have a new deployment since `trustpass` is a local variable.
That's why we have to generate it once (as a global variable) and update only if cluster deployment exists

Reply:
I see.
Thanks for the explanation